addon/model/model.ts

```diff
@@ -195,11 +196,14 @@ export default class Model {
    */
   change(
     callback: (mutator: ImmediateModelMutator) => ModelElement | void,
-    writeBack = true
+    writeBack = true,
+    saveSnapshot = true
   ) {
     const mutator = new ImmediateModelMutator(this._eventBus);
     callback(mutator);
-
+    if (saveSnapshot) {
```

Comment:
not sure if it's best before or after writing the tree (or if it even matters at all)